### DISCUSSION TITLE
Downloading local products referenced in a custom STAC catalog

### DISCUSSION THREAD

**Discussion by remi-braun** [CS-SI/eodag#454] [Q&A]
Hello,

Is there currently a way in EODAG to download (i.e. copy in this case) a product from a STAC catalog that is built with local products (ie. previously downloaded data stored on disk) ?

The query works fine, but I haven't found a way for the download 😄 

The goal is to reproduce the normal behaviour of EODAG, but checking first if products exist on disk before downloading from AWS for example.

**Comment by sbrunato** [comment]
Hello @remi-braun , no we do not have any published download plugin that would copy local products or create a symlink to them. This could be a useful feature!

For having a fallback mechanism with classic AWS download, this could be done following this feature request https://github.com/CS-SI/eodag/issues/136

**Reply by sbrunato** [reply]
I just opened a feature request for the local product download plugin https://github.com/CS-SI/eodag/issues/457
